app/models/concerns/searchable.rb

```diff
@@ -30,4 +34,10 @@ def as_indexed_json(options = {})
       as_json(self.class.indexed_json_structure)
     end
   end
+
+  def self.classes
+    ActiveRecord::Base.descendants.select { |klass|
+      klass.ancestors.include?(Searchable)
+    }
```

Comment:
Let's use do/end for multiline blocks, and reserve multiline { } use only for hash structures. Ok?